build-system/default-pre-push

```diff
@@ -25,7 +25,7 @@
 GREEN() { echo -e "\033[0;32m$1\033[0m"; }
 CYAN() { echo -e "\033[0;36m$1\033[0m"; }
 
-GULP_BUNDLE_SIZE="gulp bundle-size"
+GULP_BUNDLE_SIZE="gulp bundle-size --on_pre_push_build"
```

Comment:
I initially added this to the pre-push hook because it was an instant local-only check that would take ~1 second to run. (Pre-push hooks should be fast.)

Now that we're doing more complicated stuff, should we consider removing this check from the pre-push hook? I'm not sure the check is designed to run from the developer machines.

WDYT?

Reply:
I'd be totally cool with removing it altogether. =)

Reply:
Great. In that case, the only change needed is to delete this line :)